Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG ruby_version=2.6
+FROM ruby:${ruby_version}
+
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /opt/valkyrie
+
+COPY . .
+
+RUN chmod -R 0777 .
+
+USER 1001:0
```

Comment:
these lines cause me to get a permission error when i try to run 
```
docker-compose exec valkyrie bundle exec rake
```
and since this isn't a production image i don't think we need to set a non-default user.  this change got tests running for me (although i got a number of failures which i haven't looked into yet):
```diff 
 Dockerfile | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/Dockerfile b/Dockerfile
index 24cb7f6..d4b44dd 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -5,12 +5,8 @@ SHELL ["/bin/bash", "-c"]

 WORKDIR /opt/valkyrie

-COPY . .
-
-RUN chmod -R 0777 .
+RUN gem install bundler

-USER 1001:0
-
-ENV HOME=/opt/valkyrie
+COPY . .

-RUN gem install bundler && bundle install
+RUN bundle install --jobs "$(nproc)"
```